var/spack/repos/builtin/packages/qt/package.py

```diff
@@ -132,16 +138,30 @@ def configure(self):
     def configure(self):
         configure('-fast',
                   '-no-webkit',
+                  # XXX(osx): sdk stuff; should depend on the architecture and the SDKs available.
+                  '-arch', 'x86_64',
+                  '-sdk', '/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.9.sdk',
+                  # XXX(osx): When using Xcode's clang rather than a GCC.
+                  '-platform', 'unsupported/macx-clang',
                   *self.common_config_args)
 
 
-    @when('@5')
+    @when('@5:5.5')
     def configure(self):
-        configure('-no-eglfs',
-                  '-no-directfb',
-                  '-qt-xcb',
+        configure(#'-no-eglfs',
+                  #'-no-directfb',
+                  #'-qt-xcb',
+                  #'-arch', 'x86_64',
+                  #'-sdk', 'macosx10.11',
                   # If someone wants to get a webkit build working, be my guest!
                   '-skip', 'qtwebkit',
+                  '-skip', 'qtwebengine',
+                  *self.common_config_args)
+
+
+    @when('@5.6')
```

Comment:
Yes, thanks.  Will update it now.

On Thu, Apr 21, 2016 at 4:12 PM, Ben Boeckel <notifications@github.com>
wrote:

> In var/spack/repos/builtin/packages/qt/package.py
> <https://github.com/LLNL/spack/pull/820#discussion_r60647301>:
>
> >                    # If someone wants to get a webkit build working, be my guest!
> >                    '-skip', 'qtwebkit',
> > +                  '-skip', 'qtwebengine',
> > +                  *self.common_config_args)
> > +
> > +
> > +    @when('@5.6')
>
> Should this be @5.6:?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/LLNL/spack/pull/820/files/861f080519b8b57748d2c4dc636f2533497f3506#r60647301>
>



-- 
BEST Regards,

--
Dhanannjay (D'Jay) Deo

[ नासतो विद्यते भावो नाभावो विद्यते सतः ।
  Nothing unreal exists, nothing real can be threatened |
  (Bhagawad Geeta 2:16) ]